client/components/tinymce/index.jsx

```diff
@@ -186,11 +188,25 @@ module.exports = React.createClass( {
 
 		this.bindEditorEvents( prevProps );
 
+		if ( this.props.postCategory !== prevProps.postCategory ) {
+			console.log("this.props.postCategory = ", this.props.postCategory);
+			this.updateBodyClass();
+		}
+
 		if ( this.props.mode !== prevProps.mode ) {
 			this.toggleEditor();
 		}
 	},
 
+	updateBodyClass: function() {
+		let postCategory = this.props.postCategory;
+		let categoriesAsClasses = postCategory ? Object.keys(postCategory).map(function(category) {
+			return category.replace(/\s+/g, '-').toLowerCase();
+		}).join(' ') : '';
+		let body = this._editor.getDoc().querySelector("body");
+		body.className = classnames(body.classList, categoriesAsClasses);
```

Comment:
need to fix this line: `body.className = classnames(body.classList, categoriesAsClasses);`
There are already classed assigned to `<body>` that's unrelated to Categories. We need to find another way to do this so each `class` gets toggled properly.